### PR TITLE
Move SWIG interfaces from python to ruby

### PR DESCRIPTION
**Pull request by ahcorde**
Signed-off-by: ahcorde <ahcorde@gmail.com>

# 🎉 New feature

## Summary

This PR removes SWIG python interfaces

Require:
 - https://github.com/ignitionrobotics/ign-math/pull/343
 - https://github.com/ignitionrobotics/ign-math/pull/351

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [x] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: src/python/SphericalCoordinates_TEST.py [hunk 1]
```diff
@@ -1,562 +0,0 @@
-# Copyright (C) 2021 Open Source Robotics Foundation
```
Comment (chapulina) [inline src/python/SphericalCoordinates_TEST.py:1]: I think we're missing the migration of this class to pybind11

**Review by chapulina — CHANGES_REQUESTED** [review]
I think we're just missing the `SphericalCoordinates` migration to pybind11.

Adding a cross-reference link to the issue about enabling these SWIG files:

* https://github.com/ignitionrobotics/ign-math/issues/302

**Review by chapulina — APPROVED** [review]
:heavy_check_mark: That was an impressive number of PRs merged in a short time :smile:

**Comment by chapulina** [comment]
I think we can dismiss the ABI checker, since we're not promising ABI stability of the python shared libraries.

**Comment by osrf-triage** [comment]
This pull request has been mentioned on **Gazebo Community**. There might be relevant details there:

https://community.gazebosim.org/t/new-ignition-releases-2022-03-01-citadel-edifice-fortress/1313/1

**Pull request merged**